x-pack/plugins/siem/public/components/open_timeline/timelines_table/extended_columns.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+
+import { getEmptyValue } from '../../empty_value';
+
+import * as i18n from '../translations';
+
+/**
+ * Returns the expanded columns that are specific to the `All Timelines` view
+ * of the `Timelines` page
+ */
+export const getExtendedColumns = () => [
+  {
+    dataType: 'string',
+    field: 'updatedBy',
+    name: i18n.MODIFIED_BY,
+    render: (updatedBy: string) => (
+      <div data-test-subj="username">{updatedBy != null ? updatedBy : getEmptyValue()}</div>
```

Comment:
There is a handy helper created a while back you can use here called, `defaultToEmptyTag`, it works like this:

```ts
<div data-test-subj="username">defaultToEmptyTag(updatedBy)</div>
```

And it will perform not only the check against a `null` but also if for some reason `updatedBy` is an empty string it will add the EmptyString placeholder.